pipeline/types/analytics.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+from typing import Any, Dict
+
+
+@dataclass
+class AnalyticsEvent:
+    event_type: str
+    data: Dict[str, Any]
+    timestamp: int
```

Comment:
I left a comment on the thread where you asked about this, but let's make timestamp a `datetime.datetime` object.

Reply:
Sounds good! Added in latest commit.